### ISSUE TITLE
Error in results

### ISSUE THREAD

**Issue by MOWaelDA** [cekbote/optimal_uav_placement_and_communication#3]
hello,

kindly note that the result for random placement in your code is better than optimal placement

**Comment by cekbote** [comment]
I don't think that is possible (unless you got lucky with some seed). Can you change the parameters once and confirm?
